tests/fixtures/requests.js

```diff
@@ -0,0 +1,28 @@
+var nock = require('nock'),
+    fixtureSet = require('./fixture-set');
+
+
+// requiring nock causes it to disallow all network connections,
+// we don't want that
+nock.enableNetConnect();
+
+
+module.exports = fixtureSet({
+  setup: function(fixtures) {
+    fixtures
+      .add('stellarApi:url', 'https://test.stellar.org:9002')
+
+      .add('stellarApi:issuer', 'gBAde4mkDijZatAdNhBzCsuC7GP4MzhA3B')
+
+      .add('stellarApi:payment:str', nock(fixtures('stellarApi:url'))
+        .post('/', {"method":"submit","params":[{"secret":"sfSU4rF2UuKTCQG5kTukEqNsuUgUTgf2UuC2gsEokxtypwCeJbZ","tx_json":{"TransactionType":"Payment","Account":"gajc65uAsK4AgQM771SqkntWyVQL48GBjK","Destination":"gLk2FP9PFw1TFAt7kyAVuJpr4vLapiVRSJ","Amount":1}}]})
```

Comment:
The secret belongs to a test account I created on the old stellar test network. I asked the friendly robot at https://www.stellar.org/ to give me some credits. So this is real in that the address exists in the test stellar network's ledger, but that's pretty much it (no real currency or personal information that can be lost or anything).